trading_calendars/exchange_calendar_bvmf.py

```diff
@@ -165,7 +179,8 @@ class BVMFExchangeCalendar(TradingCalendar):
     - Proclamation of the Republic (November 15)
     - Day of Black Awareness (November 20 after 2004)
     - Christmas (December 24 and 25)
-    - Day before New Year's Eve (December 30 if NYE falls on a Saturday)
+    - Day before New Year's Eve (December 30 if NYE falls on a Saturday,
+        December 29 if NYE falls on a Monday)
     - New Year's Eve (December 31)
```

Comment:
As I mentioned with the holiday definitions, I think we could probably combine this with the above, the day before is an observance, not a separate holiday. Another way to phrase this would be "Business Day before New Year's Day".

Reply:
good idea, done